mbzirc_ign/models/sensors/mbzirc_rf_long_range/model.sdf

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0"?>
+<sdf version="1.9">
+  <model name="mbzirc_rf_long_range">
+    <link name="sensor_link">
+      <inertial>
+        <mass>0.005</mass>
+        <inertia>
+          <ixx>8.33e-06</ixx>
+          <ixy>0</ixy>
+          <ixz>0</ixz>
+          <iyy>8.33e-06</iyy>
+          <iyz>0</iyz>
+          <izz>8.33e-06</izz>
+        </inertia>
+      </inertial>
+    </link>
+    <frame name="mount_point"/>
+    <plugin
+      filename="libRFRange.so"
+      name="ignition::gazebo::systems::RFRangeSensor">
+      <!-- this overrides default RF Range system settings in the world -->
+      <!-- todo(iche033) check if param values are reasonable -->
+      <range_config>
+        <max_range>500000.0</max_range>
+        <fading_exponent>2.6</fading_exponent>
+        <l0>40</l0>
+        <sigma>20.0</sigma>
+        <rssi_1>-15</rssi_1>
+      </range_config>
+      <radio_config>
+        <tx_power>30</tx_power>
```

Comment:
@caguero I'll need your input on what good params to use here for long range RF sensor.

Reply:
Parameters adjusted in [db7ac82](https://github.com/osrf/mbzirc/pull/155/commits/db7ac820d4c4142542a9f9d734ac0706209cbe24).

The regular RF range sensor should have a range of about 500m. The RF long range sensor should have a range of about 1000m. The error should go around +-5m for the regular sensor and +-20m for the long range approximately.

Reply:
> Parameters adjusted in [db7ac82](https://github.com/osrf/mbzirc/pull/155/commits/db7ac820d4c4142542a9f9d734ac0706209cbe24).
> 
> The regular RF range sensor should have a range of about 500m. The RF long range sensor should have a range of about 1000m. The error should go around +-5m for the regular sensor and +-20m for the long range approximately.

thanks!